components/myforms/Tabs/TabPanel.tsx

```diff
@@ -11,13 +11,8 @@ export const TabPanel = (props: TabPanelProps): React.ReactElement => {
   const { children, id, labeledbyId, isActive = false } = props;
 
   return (
```

Comment:
After some discussion with a friend, the section element should only really be used if there is a heading element contained within. I would argue that if you associate a heading label that works to but most articles/* disagree with me, so best to go with a div element.